### ISSUE TITLE
fails to download large files

### ISSUE THREAD

**Issue by arscyper** [mygrocerydeals/gradle-s3-plugin#2]
setup was done using:
```
s3 {
  region = 'us-east-1'
}

task downlaodData(type: S3Download) {
    bucket = 'my-bucket
    key = "data.tar.tbz2"
    file = "${buildDir}/data/data.tar.tbz2"
}
```

error:

"Not all bytes were read from the S3ObjectInputStream, aborting HTTP connection. This is likely an error and may result in sub-optimal behavior. Request only the bytes you need via a ranged GET or drain the input stream after use."

**Comment by chubbard** [comment]
I see this error too.  And I also see java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@6e8a9af4 rejected which can happen when too many requests are made to the SDK until they supposedly fixed it in 2.2.114.  I noticed that this plugin is using a really old version 1.1.x.  So I think just upgrading the library might fix these issues.

**Comment by peter-thomas-mgd** [comment]
Fixed in version 1.1.4. Thanks @chubbard !
